Source/Engine/Core/Math/BoundingBox.cs

```diff
@@ -414,6 +414,31 @@ public static BoundingBox Merge(BoundingBox value1, BoundingBox value2)
             return box;
         }
 
+        /// <summary>
+        /// Merges the box with a point.
+        /// </summary>
+        /// <param name="point">The point to add to the box bounds.</param>
+        public BoundingBox Merge(Vector3 point)
+        {
+            BoundingBox result;
+            Vector3.Min(ref Minimum, ref point, out result.Minimum);
+            Vector3.Max(ref Maximum, ref point, out result.Maximum);
+            return result;
+        }
+
+        /// <summary>
+        /// Creates the bounding box that is offseted by the given vector. Adds the offset value to minimum and maximum points.
+        /// </summary>
+        /// <param name="offset">The offset.</param>
+        /// <returns>The result.</returns>
+        public BoundingBox MakeOffsetted(Vector3 offset)
```

Comment:
Those are taken from the C++ side 👍  https://github.com/FlaxEngine/FlaxEngine/blob/7c9e170a40f748bc2738d6ab40649fbf13c742f9/Source/Engine/Core/Math/BoundingBox.cpp#L58